### PR TITLE
Add glass

### PR DESCRIPTION
**Pull request by hhy3**

### CODE REVIEW

File: .github/workflows/benchmarks.yml [hunk 1]
```diff
@@ -39,6 +39,7 @@ jobs:
           - hnswlib
           - luceneknn
           - milvus
+          - glass
```
Comment (erikbern) [inline .github/workflows/benchmarks.yml:42]: can you keep this in alphabetical order? thanks

**Comment by erikbern** [comment]
Thanks – is the source code available online somewhere? The Git repo that you linked to only has a precompiled wheel in it

File: .github/workflows/benchmarks.yml (hunk 1 above)
Comment (hhy3) [reply .github/workflows/benchmarks.yml]: Sure

**Comment by hhy3** [comment]
> Thanks – is the source code available online somewhere? The Git repo that you linked to only has a precompiled wheel in it

I'm sorry that this is only an experimental project now and closed source. But it'll be open-source someday when it's ready.

**Comment by erikbern** [comment]
> I'm sorry that this is only an experimental project now and closed source. But it'll be open-source someday when it's ready.

Ok – let's wait until it's open source. I don't think we should include closed-source libraries in ann-benchmarks

**Comment by hhy3** [comment]
> > I'm sorry that this is only an experimental project now and closed source. But it'll be open-source someday when it's ready.
> 
> Ok – let's wait until it's open source. I don't think we should include closed-source libraries in ann-benchmarks

Ok

**Comment by hhy3** [comment]
> > I'm sorry that this is only an experimental project now and closed source. But it'll be open-source someday when it's ready.
> 
> Ok – let's wait until it's open source. I don't think we should include closed-source libraries in ann-benchmarks

Could you please tell me when the main page will be updated? So that I can catch up on time

**Pull request closed without merging**